tests/test_options.py

```diff
@@ -52,3 +53,15 @@ def test_https_options_removes_cors():
     assert https_options.cors.cors_origins == "*", "cors options were not set"
     https_options_dict = https_options._asdict_with_global_options()
     assert "cors" not in https_options_dict, "'cors' key should not exist in dict"
+
+
+def test_options_asdict_uses_cel_representation():
+    """
+    Test Param or Expression option values are converted to their
+    CEL values for manifest representation.
+    """
+    int_param = params.IntParam("min")
```

Comment:
Nit: We should also have a check to throw an error if name is not in ALL_CAPS because these will be env var names and Run only supports ALL_CAPS vars